doc/usage.md

```diff
@@ -239,7 +276,7 @@ curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d
   "@iot.id": 1,
   "@iot.selfLink": "http://localhost:8080/v1.0/Observations(1)",
   "Datastream@iot.navigationLink": "http://localhost:8080/v1.0/Observations(1)/Datastream",
-  "FeaturesOfInterest@iot.navigationLink": "http://localhost:8080/v1.0/Observations(1)/FeaturesOfInterest",
+  "FeatureOfInterest@iot.navigationLink": "http://localhost:8080/v1.0/Observations(1)/FeaturesOfInterest(2)",
   "phenomenonTime": "2016-11-18T11:04:15.790Z",
   "resultTime": "2016-11-18T11:04:15.790Z",
   "result": 12.4,
```

Comment:
Could you add the FeatureOfInterest mandatory param to the Observations requests of the deep insert section as well, please?